gflownet/policy/base.py

```diff
@@ -21,82 +20,18 @@ def __init__(self, config, env, device, float_precision, base=None):
         self.base = base
 
         self.parse_config(config)
+        self.instantiate()
 
     def parse_config(self, config):
```

Comment:
I would suggest to get rid of the method `parse_config` and the `config` argument itself in the `__init__` method and instead make all the arguments explicit (as it is done for the environments)

Reply:
Why would you want this to be removed out of curiosity? are you referring the whole `def parse_config` function to be removed from the base? in the above comment you mention `super().parse_config should be called to parse ...` if we remove then i don't get which parse_config we will be initializing. Have a look at my next commit for minor update tho.

Reply:
pushed update on this :)